pallets/genetic-analysts/src/lib.rs

```diff
@@ -88,9 +88,12 @@ where
 	pub stake_status: StakeStatus,
 	pub verification_status: VerificationStatus,
 	pub availability_status: AvailabilityStatus,
+	pub unstake_at: Moment,
```

Comment:
should we create storage migration for this changes on struct?

Reply:
I don't think there's a lot of genetic analysts yet.